app/api/api/views.py

```diff
@@ -616,7 +617,7 @@ def get_queryset(self):
 
 
 class DatasetCreateView(generics.CreateAPIView):
-    serializer_class = DatasetViewSerializer
```

Comment:
Could we do this the other way around? 

As in this view - the "legacy" form should still use `DatasetViewSerializer`

The form you had to update the serializer for should use `DatasetViewSerializerV2`

`V2` should be anything we upgrade